docs/internal_api.yml

```diff
@@ -1,193 +1,193 @@
-swagger: '2.0'
+openapi: 3.0.3
+
 info:
-  title: Device Connect Internal API
-  version: '1'
+  title: Device Connect
   description: |
-    Internal API of device connect service
-
-host: 'mender-deviceconnect:8080'
-basePath: '/api/internal/v1/deviceconnect'
-schemes:
-  - http
-
-produces:
-  - application/json
-consumes:
-  - application/json
-
-responses:
-  InternalServerError: # 500
-    description: Internal Server Error.
-    schema:
-      $ref: "#/definitions/Error"
-  InvalidRequestError: # 400
-    description: Invalid Request.
-    schema:
-      $ref: "#/definitions/Error"
+    Internal API for managing persistent device connections.
+    Intended for use by the web GUI.
+  version: "1"
+
+  contact:
+    name: Mender Support
+    email: support@mender.io
+    url: https://mender.io/contact-us
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: https://hosted.mender.io/api/internal/v1/deviceconnect
```

Comment:
This URL is wrong, we do not expose internal APIs over hosted mender. 
As of now, we used `http://mender-deviceconnect:8000/api/internal/v1/deviceconnect` or similar.
Why are you using hosted.mender.io here?

Reply:
This followed through from the copy-paste when I initialized these docs.